client.go

```diff
@@ -1104,6 +1104,14 @@ func (c *Client) handlePongMessage(msg PongMessage) {
 	}
 }
 
+func (c *Client) RestrictChatForFollowers(channel string, duration string) {
```

Comment:
Should be called FollowersOn for less confusion

Reply:
okay